frontend/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@shopgate-project/product-image-badges",
-  "version": "1.4.0",
   "description": "Extension will allow a user to display a image badge on a product image based on tag or property.",
```

Comment:
in my setup the linter didn't work. updating eslint inside the devDependencies to `^6.8.0` worked for me